### DISCUSSION TITLE
Broken docker config json with newest flux

### DISCUSSION THREAD

**Discussion by pefeigl** [fluxcd/flux2#2745] [Q&A]
Hello!

We've updated from 0.29.x to 0.30.2, and are seeing a strange bug:
All `kubernetes.io/dockerconfigjson` type secrets (SOPS-encrypted) are generated incorrectly, the *content* of key `.dockerconfigjson` is as follows:
```
{
	"data": "{\n\t\"auths\": {\n\t\t\"example.com\": {\n\t\t\t\"auth\": \"abcdefghijklmnopqrstuvwxyz...\"\n\t\t}\n\t}\n}"
}
```
The correct data is in there, however there's an extra `data` added around it, which breaks things..

We have verified that the encrypted data inside SOPS is correct, it does not contain the extraneous `data` wrapper.

Is this related to https://github.com/fluxcd/kustomize-controller/pull/644/files#diff-fc2e7672461dad3c1054ca02c6da5f60231da6d8d8e0c588f1ee24edd184a3a6R851 and if yes, how do we best fix this? Is there a way to *downgrade* flux?
`flux install -v v0.29.5` does not want to run.

Thanks for any help with this :-/

**Comment by kingdonb** [comment]
Can you check out this thread please:

* https://github.com/fluxcd/kustomize-controller/issues/654
